### PR TITLE
Restore card callout

### PR DESCRIPTION
**Pull request by jwoolley**
The card you're thinking of is https://goodgamery.com/api/mtg/image?card=Wild+Guess

### CODE REVIEW

**Comment by jwoolley** [comment]
Whoops, I branched off of the latest decktech-image changes, so this PR has those changes too and should be merged first.

File: src/data/grammar/grammar.yaml [hunk 1]
```diff
@@ -18,7 +18,8 @@
       	"#gathererRulingImage#",
         "#inGraphics#",
         "#mtgComic#",
-        "#mtgDeckTechGraphic#"
+        "#mtgDeckTechGraphic#",
+        "cardCallout"
```
Comment (mdiehr) [inline src/data/grammar/grammar.yaml]: Needs the ##

Comment (jwoolley) [inline src/data/grammar/grammar.yaml]: WAWAWA

**Comment by jwoolley** [comment]
I just rebased, so I'll cancel this PR and resubmit.

**Comment by jwoolley** [comment]
> I just rebased, so I'll cancel this PR and resubmit.

**Pull request closed without merging**